### PR TITLE
Movie widgets imported from own library

### PR DESCRIPTION
**Pull request by HoracioDuca**
### The Trending Movies App implementing own Widget's library

• In case of _allMovies_ were null, now has a checkpoint that handles that value.

• **movies_widgets library** is imported with my own widgets.

image

### CODE REVIEW

**Review by marianosalvetti — APPROVED** [review]

**Review by marianosalvetti — APPROVED** [review]

File: lib/src/models/movie.dart [hunk 1]
```diff
@@ -16,10 +16,15 @@ class Movie {
   factory Movie.fromJson(
     Map<String, dynamic> parsedJson,
   ) {
-    var allMovies = parsedJson['results'] as List;
-
-    List<ResultMovie> movieTemp =
-        allMovies.map((eachMovie) => ResultMovie.fromJson(eachMovie)).toList();
+    List<ResultMovie> movieTemp;
+    if (parsedJson['results'] != null) {
+      var allMovies = parsedJson['results'] as List;
```
Comment (luisjavierglb) [inline lib/src/models/movie.dart:22]: Doing this without wrapping on a try catch can cause unwanted and unhanlded exceptions on the code

**Review by luisjavierglb — APPROVED** [review]
LGTM

File: lib/src/models/movie.dart (hunk 1 above)
Comment (HoracioDuca) [reply lib/src/models/movie.dart:22]: Done! In case of the movies list callback were empty now its handled by a try/catch exception. Thank you Javi.

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**